screens/ExpenseTrackerScreen.js

```diff
@@ -95,12 +112,37 @@ export default function ExpenseTrackerScreen() {
 }
 
 // eslint-disable-next-line react/prop-types
-function Item({ title, amount }) {
+function Item({ id, title, amount, removeExpense }) {
+  const [highlighted, setHighlighted] = useState(false);
+
+  const highlightItem = () => {
+    setHighlighted(true);
+  };
+
+  const notHighlightItem = () => {
+    setHighlighted(false);
+  };
+
+  if (highlighted) {
+    return (
+      <View style={[styles.item, styles.highlighted]}>
+        <TouchableOpacity onPress={() => removeExpense(id)}>
+          <Text style={styles.big}>Remove</Text>
+        </TouchableOpacity>
+        <TouchableOpacity onPress={notHighlightItem}>
+          <Text style={styles.small}>Cancel</Text>
+        </TouchableOpacity>
+      </View>
+    );
+  }
+
   return (
-    <View style={styles.item}>
-      <Text style={styles.small}>{title}</Text>
-      <Text style={styles.big}>{amount}</Text>
-    </View>
+    <TouchableNativeFeedback onLongPress={highlightItem}>
+      <View style={styles.item}>
+        <Text style={styles.small}>{title}</Text>
+        <Text style={styles.big}>{amount}</Text>
+      </View>
+    </TouchableNativeFeedback>
   );
 }
 
```

Comment:
<img width="146" alt="Screenshot 2020-05-15 at 16 13 13" src="https://user-images.githubusercontent.com/43641321/82060570-dab25500-96c7-11ea-8ac3-40c3c9dfecb6.png">
 this is what i get on simulator and iphon

Reply:
* A wrapper for making views respond properly to touches (Android only) https://reactnative.dev/docs/touchablenativefeedback 
I guess a condition would help ?

Reply:
I updated Expo to 38 and iOS makes no problems. It seems that with a newer React Native version the Android only for `TouchableNativeFeedback` is removed and some iOS fallback is added.